### ISSUE TITLE
Submit GET forms to the plugin adminstration

### ISSUE THREAD

**Issue by cmb69** [cmsimple-xh/cmsimple-xh#66]
[Forum discussion](http://cmsimpleforum.com/viewtopic.php?t=7820)

**Comment by cmb69** [comment]
As there is no easy resolution for this issue, and it's easily possible to work around it (see cmb69/realblog_xh@e12f2ab0be52cfbbe66d203d745fb7056a9fab08), I suggest to postpone it to 1.8.

**Comment by cmb69** [comment]
Well, actually this issue has already solved been solved by solving #19. POC:
````php
<form action="<?=$sn?>">
    <input type="hidden" name="selected" value="pagemanager">
</form>
````
Closing as duplicate.
